### PR TITLE
Update release notes for 0.146

### PR DESCRIPTION
**Pull request by haozhun**

### CODE REVIEW

File: presto-docs/src/main/sphinx/release/release-0.146.rst [hunk 1]
```diff
@@ -13,6 +13,8 @@ Hive Changes
 * Fix incorrect skipping of data in Parquet during predicate push-down.
 * Fix reading of Parquet maps and lists containing nulls.
 * Fix reading empty ORC file with ``hive.orc.use-column-names`` enabled.
+* Fix unexpected exception from output stage JSON truncation in completion event
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.146.rst]: This seems to be missing something here: "from output stage JSON trunction"

Or try to rephrase this. Also, end release notes in a period.

File: presto-docs/src/main/sphinx/release/release-0.146.rst [hunk 2]
```diff
@@ -13,6 +13,8 @@ Hive Changes
...
+* Fix writing to S3 filesystem when the staging directory is a symlink to a directory
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.146.rst]: Good, but add period

File: presto-docs/src/main/sphinx/release/release-0.146.rst (hunk 1 above)
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.146.rst]: This should be under General, not Hive

**Comment by electrum** [comment]
I'm removing the first item because it only affects users that use a modified Presto with a custom event client and it's mostly an internal change for debugging.

**Comment by electrum** [comment]
Merged, thanks!

**Comment by haozhun** [comment]
But it can mean missing log entries, and I believe the users do care.

**Pull request closed without merging**